src/app/hooks/useGetTokenPrice.tsx

```diff
@@ -0,0 +1,29 @@
+import { bignumber } from 'mathjs';
+import { useMemo } from 'react';
+import { Asset } from 'types';
+import { APOLLO_POLL_INTERVAL } from 'utils/classifiers';
+import { AssetsDictionary } from 'utils/dictionaries/assets-dictionary';
+import { useGetTokenQuery } from 'utils/graphql/rsk/generated';
+
+export const useGetTokenPrice = (asset: Asset, type: 'BTC' | 'USD' = 'BTC') => {
```

Comment:
I don't like `type` typing - pls extract it to an enum that can stay in this file (you can call it `DenominationAsset`).